deploy/crds/keda.sh_v1alpha1_scaledjob_cr.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: keda.sh/v1alpha1
```

Comment:
Just wondering when we move them from alpha to v1?

Reply:
It's up for discussion, but `v1alpha1` convention allow us to do even breaking changes in that spec. So I think that we can move forward to `v1beta1` or `v1` some time after we are out with v2 and are sure that the spec is what we want to have.

Reply:
That's fine by me!